AsyncDisplayKit/ASMapNode.mm

```diff
@@ -265,17 +277,23 @@ - (void)addLiveMap
     [_mapView addAnnotations:_annotations];
     [weakSelf setNeedsLayout];
     [weakSelf.view addSubview:_mapView];
-    
-    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
-      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
+
+    if (self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsZoomed) {
+      BOOL const animated = self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsAnimated;
+      [_mapView showAnnotations:_mapView.annotations animated:animated];
     }
+    
+//    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
+//      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
+//    }
   }
 }
 
 - (void)removeLiveMap
 {
+  self.userInteractionEnabled = false;
```

Comment:
@appleguy You're right, sorry, I've been working with swift a lot lately :)

@Adlai-Holler @appleguy how about if I hold the previous state in some temporary variable and then restore it afterwards?
Because in the state it is now, we are changing the state of self.userInteractionEnabled. The reason why I placed that in removeLiveMap was to go back to its original state and thus not create unexpected behavior.

Imagine this scenario:
1- Create ASMapNode (`userInteractionEnabled = YES`, based on what the discussion is now)
2- Set `mapNode.userInteractionEnabled = NO` for whatever reason the user decides to do that
3- Toggle `mapNode.liveMap = YES` (this triggers `self.userInteractionEnabled = YES`)
4- Done with live map, go back to image. `mapNode.liveMap = NO` (current state is: `userInteractionEnabled = YES`)

In this case we changed the state of `userInteractionEnabled`